src/views/BreadView/AddBreadDialog.js

```diff
@@ -0,0 +1,61 @@
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import FastfoodIcon from '@material-ui/icons/Fastfood';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import ListItemText from '@material-ui/core/ListItemText';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+
+class AddBreadDialog extends React.Component {
+  handleClose = () => {
+    this.props.onClose(this.props.selectedValue);
+  };
+
+  handleListItemClick = value => {
+    this.props.onSelect(value);
+  };
+
+  render() {
+    const {classes, onClose, selectedValue, items, ...other } = this.props;
+
+    return (
+      <Dialog onClose={this.handleClose} aria-labelledby="dialog-title" {...other}>
+        <DialogTitle id="dialog-title">Kies een brood</DialogTitle>
+        <div style={{minWidth: '250px'}}>
```

Comment:
die div mag weg, maar zet dan <Dialog maxWidth="xs" fullWidth> bv. (zie docs)

Reply:
welp, die comment is precies getruncated ...
er stond: die div mag weg, maar zet dan: <Dialog maxWidth="xs" fullwidth> (zie https://material-ui.com/api/dialog/)

Reply:
aha! omdat ik < gebruikte

Reply:
Ziet er goed uit!